data/validation_schemas/HTAN_validation_schema_FamilyHistory.json

```diff
@@ -0,0 +1,518 @@
+{
```

Comment:
Out of curiosity, what's the purpose of this file?

Reply:
This one will be used in a PR I will issue once this is merged. We are adding a new method called `submit_manifest()` which has new validation utilities and this file will be used by that (in an example).